homeassistant/components/updater/__init__.py

```diff
@@ -16,6 +16,7 @@
 
 ATTR_RELEASE_NOTES = "release_notes"
 ATTR_NEWEST_VERSION = "newest_version"
+ATTR_CURRENT_VERSION = "current_version"
```

Comment:
Would it be clearer if the attribute was named "local_version" or "installed_version"?

Reply:
Sure, I can update it to installed_version.